core/include/drivers/semihosting_console.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2024 Andes Technology Corporation
+ */
+#ifndef __DRIVERS_SEMIHOSTING_CONSOLE_H
+#define __DRIVERS_SEMIHOSTING_CONSOLE_H
+
+#include <drivers/serial.h>
+
+/*
+ * struct semihosting_console_data - Structure for semihosting console driver
+ * @chip - General structure for each serial chip
+ * @file_path - Path of the file on the semihosting host system
+ * @fd - Handle of the file at @file_path
```

Comment:
`... or -1 if using the semihosting console`

Reply:
Added into https://github.com/OP-TEE/optee_os/pull/6700/commits/f6bb8ab6267f21b773cd9cc8e48fa377f8288e48